src/if-run/builtins/copy-param/README.md

```diff
@@ -54,6 +54,15 @@ The `parameter-metadata` section contains information about `description`, `unit
   - `unit`: unit of the parameter
   - `aggregation-method`: aggregation method of the parameter (it can be `sum`, `avg` or `none`)
 
+### Mapping
+
+The `mapping` block allows to rename the parameters of the input and output with new names. The structure of the `mapping` block is:
+
+```yaml
+mapping:
+  'old-name': 'new-name'
```

Comment:
its not clear where the mapping should be. f.ex. in plugin settings or in context

Reply:
yeah, you're right. In this case, I think it will be good to remove from the plugin's readme file